_pages/about-us/teams/products-and-platforms.md

```diff
@@ -15,13 +17,13 @@ Products and Platforms is one of the 5 business units within 18F. We strive to i
 
 * 18F's work on the [eRegs platform](https://eregs.github.io/) with various partners.
 
-* With partners in TTS, the [Draft US Web Design Standards](https://standards.usa.gov/): open-source UI components and visual styles to create beautiful, consistent experiences across federal government websites.
```

Comment:
One thought I had: Will all readers know what _TTS_ stands for? If not, I'd recommending using the full name (followed by the acronym in parentheses) on the first reference, and then using the acronym on all subsequent references — this to ensure that folks aren't confused!